sensors/icinga2_state_change_events.py

```diff
@@ -60,8 +60,12 @@ def process_event(self, event):
         self.logger.info('Processing event: %s', event)
         payload = {}
         event = json.loads(event)
-        payload['service'] = event['service']
-        payload['host'] = event['host']
+        if 'service' in event:
+            payload['service'] = event['service']
+            payload['check_type'] = 'service'
```

Comment:
@atl-tortue, we can add the host line in the service block and it will ensure backward compatibility and prevent exceptions being raised for host event types.
```
        if 'service' in event:
            payload['host'] = event['host']
            payload['service'] = event['service']
            payload['check_type'] = 'service'
```